### ISSUE TITLE
Find out whether current diagrams are accessible

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-design-system#1397]
Within our content, I've seen some diagrams that I struggled to read without zooming in on. That made me wonder whether there were accessibility issues.

Two of the diagrams are on GOV.UK, and one is on GitHub. All three diagrams contain small text in boxes.

**GOV.UK**
[Patterns > Ethnic groups](https://design-system.service.gov.uk/patterns/ethnic-group/):

![image](https://user-images.githubusercontent.com/72507742/98270111-7097b000-1f86-11eb-87b8-5c973f94a506.png)

[Patterns > Check a service is suitable](https://design-system.service.gov.uk/patterns/check-a-service-is-suitable/):

![image](https://user-images.githubusercontent.com/72507742/98270152-7ab9ae80-1f86-11eb-9bb0-3f5b8316a0a3.png)

**GitHub**
[Design System flow diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams):

![image](https://user-images.githubusercontent.com/72507742/98270225-91600580-1f86-11eb-973b-3dc0d9d285f5.png)

Is this something we should explore further?

**Comment by CharlotteDowns** [comment]
## Actions
- [ ] Investigate whether the current images need a change in design or just conversion to .svgs
- [ ] Pull out existing alt text for images and make sure the text is included in the main content, add markup for decorative images <image alt="">.
- [ ] Add accessibility guidance to the [Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams/tree/5454387827b2d11d91fd266125c4905f567c91cf)

**Comment by kellylee-gds** [comment]
@CharlotteDowns are you working on this currently?

**Comment by EoinShaughnessy** [comment]
Whoops, I should have closed this ages ago - apologies.
